.gitignore

```diff
@@ -3,6 +3,7 @@
 /dist/*.ico
 /dist/*.map
 /dist/*.svg
+/dist/*.html
```

Comment:
yep, and just to provide context, we are generated an `app.go` file that embeds our bundled css and js. This `app.go` file is imported into drone and compiled into the main binary.